.github/actions/cleanup/main.go

```diff
@@ -0,0 +1,37 @@
+package main
+
+import (
+	"context"
+	"log"
+
+	"go.mongodb.org/atlas/mongodbatlas"
+)
+
+func main() {
+	config, err := NewConfig()
+	if err != nil {
+		log.Fatalf("error getting config: %s", err)
+	}
+
+	client, err := SetupClient(config.PublicKey, config.PrivateKey, config.ManagerUrl)
+	if err != nil {
+		log.Fatalf("error creating atlas client: %s", err)
+	}
+
+	projects, _, err := client.Projects.GetAllProjects(context.Background(), &mongodbatlas.ListOptions{
+		ItemsPerPage: 250,
+	})
+	if err != nil {
+		log.Fatalf("error getting projects: %s", err)
+	}
+	projectList, err := PrepareListToDelete(projects, config.DeleteAll, config.Lifetime)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	ctx := context.Background()
+	ok := DeleteProjects(ctx, client, projectList)
+	if !ok {
+		log.Printf("Not all project deleted. Please run cleaner again later")
```

Comment:
Maybe it worth to return a different exit code here?

Reply:
Done